### DISCUSSION TITLE
CYPRESS - cy.click() can only be called on a single element. Your subject contained 3 elements.

### DISCUSSION THREAD

**Discussion by thiagojosepdu** [cypress-io/cypress#21314] [CI setup] [answered]
![image](https://user-images.githubusercontent.com/42850943/166473520-86eefeb1-0608-4256-8ab2-5b471ec1ff10.png)
![image](https://user-images.githubusercontent.com/42850943/166473795-fcf1f880-1423-4e88-8a70-efbfdb2e87d2.png)
for some reason CYPRESS does not enter my line 40 showing the error in the figure below

**Comment by jjhelguero** [comment, marked as answer]
Your `.get()` returned 3 matching elements. It sounds like you were expecting one. Your custom command was executed.

**Reply by zainsadaqat** [reply]
What we can do other than using first() and last() because [0], [1], [2] doesn't work?

**Reply by Rafael-Pull** [reply]
> What we can do other than using first() and last() because [0], [1], [2] doesn't work?

.eq(2) or .eq(1) works
